pom.xml

```diff
@@ -1380,7 +1379,7 @@
          <dependency>
             <groupId>org.jboss.threads</groupId>
             <artifactId>jboss-threads</artifactId>
-            <version>${versionx.org.jboss.threads.jboss-threads}</version>
+            <version>${version.jboss.threads}</version>
```

Comment:
Should we define this in the build/configuration pom in the plugin section like some others? Honestly, I am not sure what the distinction between versionx and version is in this case.